en/docs/install-and-setup/configuring-api-manager-for-ob.md

```diff
@@ -79,36 +79,75 @@ database server, and the JDBC driver.
    for access without PSU involvement per day. This enables throttling requests according to the `frequencyPerDay` value provided 
    in accounts initiation request. For a request to be throttled, the TPP should initiate the request.
 
-    ``` toml
-    [open_banking_berlin.consent.freq_per_day]
-    enable = true
-    ```
-    
+    1. To configure the custom throttling policy, enable the `frequencyPerDay` property as follows:
+
+        ``` toml
+        [open_banking_berlin.consent.freq_per_day]
+        enable = true
+        ```
+
+    2. Deploy the throttling policy located in `<WSO2_OB_APIM_BERLIN_TOOLKIT_HOME>/repository/resources/frequencyPerDayPolicy` 
```

Comment:
if we know the file name we can directly give the file name

Reply:
Hi @isharailanga, `frequencyPerDay` is the name of the file. I couldn’t find the extension for that. 
Shall I leave it as it is?